flake.nix

```diff
@@ -56,6 +56,9 @@
       legacyPackages = forAllSystems (system: import ./. { inherit system; });
 
       nixosModules = {
+        assertions = ./nixos/modules/misc/assertions.nix;
+        lib = ./nixos/modules/misc/lib.nix;
+        meta = ./nixos/modules/misc/meta.nix;
```

Comment:
"Meta" is not a suitable scope for a module. This file has `meta.buildDocsInSandbox`, `meta.doc` and `meta.maintainers`. It should probably be split into `meta/docs.nix` and `meta/maintainers.nix`, at least, but even then, those modules aren't usable on their own either.